pyproject.toml

```diff
@@ -1,7 +1,65 @@
+[build-system]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst_butler_migrate"
```

Comment:
We tend to name the packages `lsst-butler-migrate` with hyphens. (or `lsst-daf-butler-migrate` but I guess it doesn't matter that the `daf` has disappeared).

Reply:
I need to change that, I think better to have `daf` there.